byte eye=1
byte chin=2
byte camera_type
Label[] labels